blocks/edit/prose/index.js

```diff
@@ -43,15 +43,22 @@ let lastCursorPosition = null;
 let daPreview;
 let updatePoller;
 
+const debouncedCheckForLocNodes = debounce(checkForLocNodes, 500);
+
 function dispatchTransaction(transaction) {
   if (!window.view) return;
 
   if (transaction.docChanged) {
     hasChanged += 1;
     sendUpdates = true;
   }
+
   const newState = window.view.state.apply(transaction);
   window.view.updateState(newState);
+
+  if (transaction.docChanged) {
+    debouncedCheckForLocNodes(window.view);
```

Comment:
Why do you need `debouncedCheckForLocNodes` if used only once ?

```suggestion
    debounce(window.view, 500);
```